lib/cog/commands/help.ex

```diff
@@ -75,11 +61,84 @@ defmodule Cog.Commands.Help do
       {:ok, body} ->
         {:reply, req.reply_to, body, state}
       {:error, error} ->
-        {:error, req.reply_to, error, state}
+        {:error, req.reply_to, error_msg(error), state}
+    end
+  end
+
+  # If multiple args are passed then we join with "-" before doing a lookup.
+  # This should account for users passing command names using the optional
+  # "git style" syntax.
+  defp lookup(args) when is_list(args) do
+    Enum.join(args, "-")
```

Comment:
We're trading consistency for implementation ease. Consistency is why I suggested reusing the parser but this is fine for now. If the bundle name becomes an issue then we can fix that.

Another thought is that you could add logic here to detect hyphenated bundle names and reject them.